src/workloads/execution/ExpressiveQueries.yml

```diff
@@ -0,0 +1,207 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+
+# Phases:
+# 1. Insert data
+# 2. Fsync
+# 3. Plain query (without index or projection)
+# 4. Query with projection, but without index
+# 5. Create index
+# 6. Fsync
+# 7. Query with projection and with index
+# 8. Drop collection
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1 # Loader doesn't support multiple threads for a single collection
+  Phases:
+  - Repeat: 1
+    Database: &DB test
+    Threads: 1
+    CollectionCount: 1 # Collection name will be Collection0, this is not configurable.
+    DocumentCount: 1000000 # 1M
```

Comment:
I *think* Genny supports smart numbers, so you may be able to write 1M or 1e6. Looks like 1e6 should work https://github.com/mongodb/genny/blob/ac2c06a1ec477c569ea27f818a549b38cad08e01/src/workloads/selftests/GennyOverhead.yml#L32

Reply:
I believe Genny uses stod, which is documented here https://en.cppreference.com/w/cpp/string/basic_string/stof
The best documentation in Genny seems to be the test cases today. https://github.com/mongodb/genny/blob/e8ebcbee0d5b9e99a97cdfffa75e70ed4b25078e/src/gennylib/test/conventions_test.cpp